salesloft/metadata/schemas.json

```diff
@@ -9,17 +9,20 @@
         "name": "name",
         "order": "order",
         "updated_at": "updated_at"
-      }
+      },
+      "url": "https://developers.salesloft.com/docs/api/account-stages-index"
     },
     "account_tiers": {
       "displayName": "List Account Tiers",
       "fields": {
+        "active": "active",
         "created_at": "created_at",
         "id": "id",
         "name": "name",
         "order": "order",
         "updated_at": "updated_at"
-      }
+      },
+      "url": "https://developers.salesloft.com/docs/api/account-tiers-index"
```

Comment:
This url points to documentaion on `Listing Account Tiers`
The whole file has identical change with respective doc links

Reply:
I must have missed this last time, the display names here should look like "Account Tiers" and not "List Account Tiers", since they should be the display names for the object and should look nice in the UI library as the names of the tabs. It seems like that's not available on the page (example: https://developers.salesloft.com/docs/api/account-tiers-index/), so can you revise the script to manually construct it from the object name by replacing the dashes with spaces and capitalizing the first letter of each word?

Feel free to do in follow up PR.

Reply:
I see. Yes, thats a problem.